master/buildbot/test/unit/test_changes_p4poller.py

```diff
@@ -441,7 +441,9 @@ def check(res):
         d.addCallback(check)
         return d
 
-
+    def test_resolveWho_callable(self):
+        self.assertRaisesConfigError("You need to provide a valid callable for resolvewho",
```

Comment:
you need to inherit configmixin to have this check available

Reply:
Thanks - got sidetracked in the office today and forgot to check my update PR